src/widgetastic/browser.py

```diff
@@ -751,6 +751,18 @@ def send_keys_to_focused_element(self, *keys):
         """
         ActionChains(self.selenium).send_keys(*keys).perform()
 
+    def copy(self):
+        """Select all from the current element and copy to clipboard."""
+        ActionChains(self.selenium).key_down(Keys.CONTROL).send_keys('a').key_up(
+            Keys.CONTROL).perform()
+        ActionChains(self.selenium).key_down(Keys.CONTROL).send_keys('c').key_up(
+            Keys.CONTROL).perform()
+
+    def paste(self):
+        """Paste from clipboard to current element."""
+        ActionChains(self.selenium).key_down(Keys.CONTROL).send_keys('v').key_up(
```

Comment:
I think this action should be considered a type of `send_keys`, calling `self.plugin.before_keyboard_input` and `self.plugin.after_keyboard_input` before and after performing the action chain.  That way custom browser plugins will consistently execute these methods before and after an action that is equivalent to keyboard input.